.github/workflows/osx.yml

```diff
@@ -15,46 +17,32 @@ jobs:
     steps:
       -
         name: brew packages
-        run: brew install monetdb
-      - 
+        run: brew install bison
```

Comment:
why build Jul2021? there is a binary bottle installable with homebrew, which makes the CI complete quicker.

Reply:
I missed the conversation on the slack channel, I understand now.